mne_bids/meg_bids.py

```diff
@@ -105,8 +135,21 @@ def _events_tsv(events, raw, fname, event_id, verbose):
 
 
 def _scans_tsv(raw, raw_fname, fname, verbose):
-    """Create tsv file for scans."""
+    """Create a scans.tsv file and save it.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The data as MNE-Python Raw object.
+    raw_fname : str
+        Relative path to the raw data file.
+    fname : str
+        Filename to save the channels.tsv to.
+    verbose : bool
+        Set verbose output to true or false.
 
+    """
+    # get MEASurement date from the data info
```

Comment:
measurement

Reply:
that's intentional because I finally figured out what "meas" as in "meas_date" was an abbreviation for and I thought that maybe other people would like to stumble over that. If you insist, I can change it though ;-)

Reply:
lol okay fair enough